pwdsphinx/websphinx.py

```diff
@@ -1,22 +1,23 @@
 #!/usr/bin/env python3
 #
 # This file is part of WebSphinx.
-# SPDX-FileCopyrightText:  2018 pitchfork@ctrlc.hu
-# SPDX-FileCopyrightText:  GPL-2.0-or-later
```

Comment:
As you mentioned that the preferred license in GPL v3, I update accordingly the comment header.